src/Module/RandomProfile.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Model\Contact;
+use Friendica\Model\GContact;
+
+/**
+ * Redirects to a random profile of this node
```

Comment:
Records from the `gcontact` table doesn't contain only profiles from this node. A more accurate description would be `Redirects to a random Friendica profile this node knows about`